cmd/genconversion/conversion.go

```diff
@@ -94,7 +94,7 @@ func main() {
 	generator.AddImport(path.Join(pkgBase, "api/resource"))
 	// TODO(wojtek-t): Change the overwrites to a flag.
 	generator.OverwritePackage(version, "")
-	for _, knownType := range api.Scheme.KnownTypes(version) {
+	for _, knownType := range api.Scheme.KnownTypes(strings.TrimLeft(*groupVersion, "api/")) {
```

Comment:
I'm gonna go out on a limb and say that we should never have a line like this in our code. It's too confusing. Either:
a) Put a "CanonicalizeGV()" function somewhere and use that
OR
b) (preferred) Make everyone use the same name for particular groupVersions.